topics/admin/tutorials/ansible-galaxy/tutorial.md

```diff
@@ -362,7 +372,7 @@ The configuration is quite simple thanks to the many sensible defaults that are
 >    > >   pre_tasks:
 >    > >     - name: Install Dependencies
 >    > >       package:
->    > >         name: ['git', 'make', 'python-psycopg2', 'python-virtualenv']
+>    > >         name: ['git', 'make', 'python3-psycopg2', 'python3-virtualenv']
```

Comment:
```suggestion
>    > >         name: ['git', 'make', 'python3-psycopg2', 'virtualenv']
```

Reply:
At a certain point I got this error:
```
TASK [natefoo.postgresql_objects : Create and drop users] *********************************************************************************************************************************************************
failed: [gcc-6.training.galaxyproject.eu] (item={u'name': u'galaxy'}) => {"ansible_loop_var": "item", "changed": false, "item": {"name": "galaxy"}, "msg": "Failed to import the required Python library (psycopg2) on gcc-6's Python /usr/bin/python. Please read module documentation and install in the appropriate location. If the required library is installed, but Ansible is using the wrong Python interpreter, please consult the documentation on ansible_python_interpreter"}
```
which I fixed by reverting to `python-psycopg2`